react/react2/week1/search-app/git-app/src/components/Home.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useRef, useEffect } from 'react';
+import SearchUser from './SearchUser';
+import './main.css';
+
+function Main() {
+  const [username, setUsername] = useState('');
```

Comment:
i would name it to something more related to search functionality, such as "usernameQuery" / "serachQuery" etc